frontend/src/toolbar/stats/HeatmapStats.tsx

```diff
@@ -9,15 +8,66 @@ import { LemonInput } from 'lib/components/LemonInput/LemonInput'
 import { currentPageLogic } from '~/toolbar/stats/currentPageLogic'
 import { LemonButton } from 'lib/components/LemonButton'
 import { IconSync } from 'lib/components/icons'
-import { AlertMessage } from 'lib/components/AlertMessage'
+import { LemonSwitch } from 'lib/components/LemonSwitch/LemonSwitch'
+import { Tooltip } from 'lib/components/Tooltip'
+import { toolbarButtonLogic } from '~/toolbar/button/toolbarButtonLogic'
+import './HeatmapStat.scss'
+
+/**
+ * Within the toolbar if a tooltip attaches to the body (the default) it is not visible
+ *
+ * LemonButton can accept a `disabledReason` prop, but it attaches its tooltip to the body
+ *
+ * This component wraps the LemonButton in a div, and attaches the tooltip to that div
+ * and passes the correct stacking context so that the tooltip is visible
+ * */
+function ButtonWithTooltipInStackingContext(props: {
+    canLoadMoreElementStats: any
+    onClick: () => any
+    popupContainer: () => any
+}): JSX.Element {
```

Comment:
A lot of the rest of the code uses this pattern to solve the same problem: `getPopupContainer={getShadowRootPopupContainer}`.

Can we add `getPopupContainer` as something we pass to `LemonButton`, and just set it accordingly on the tooltip?

We could also use a React context to pass this around 🤔